ql/src/test/results/clientpositive/llap/subquery_select.q.out

```diff
@@ -3943,10 +3943,14 @@ STAGE PLANS:
                 mode: mergepartial
                 outputColumnNames: _col0
                 Statistics: Num rows: 1 Data size: 4 Basic stats: COMPLETE Column stats: COMPLETE
-                Reduce Output Operator
-                  sort order: 
-                  Statistics: Num rows: 1 Data size: 4 Basic stats: COMPLETE Column stats: COMPLETE
-                  value expressions: _col0 (type: int)
+                Select Operator
```

Comment:
Did the join change type due to this?

Reply:
I don't see the join type being changed. The change only introduced an additional SELECT op. Can you point me to it?